docs/source/contributing/contributing.md

```diff
@@ -196,3 +196,16 @@ Server:
 ```
 
 Then you are good to go!
+
+## Building the documentation locally
+
+If you only changed the documentation, you can also build the documentation locally using `sphinx` .
+
+```bash
+#pip install -r docs/doc-requirements.txt
```

Comment:
Why is this commented out? My guess is that it is meant to mean "maybe you need to run this but maybe not" so I'd suggest we add a comment that says that or we just put it without a comment symbol as it should be a noop to run this command many times.

Reply:
Fair enough - re-added